os/arch/Kconfig.chip

```diff
@@ -734,23 +729,30 @@ config RAM_VSTART
 config RAM_SIZE
 	int "Primary RAM size"
 	default 0
-	help
+	---help---
 		The size in bytes of the installed primary RAM. "Primary" RAM
 		refers to the RAM that you link program code into.  If program code
 		does not execute out of RAM but from FLASH, then you may designate
 		any block of RAM as "primary."
 
+config DDR
+	bool "Use DDR memory"
+	default n
+	depends on ARCH_CHIP_BCM4390X
```

Comment:
I wonder, whether we need to create this dependency with respect to specific chip? Because any board which supports external DDR can enable this.  In my opinion default n would be sufficient .